external/storm-druid/src/main/java/org/apache/storm/druid/bolt/DruidConfig.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.storm.druid.bolt;
+
+import com.metamx.tranquility.tranquilizer.Tranquilizer;
+
+import java.io.Serializable;
+
+public class DruidConfig implements Serializable {
+
+    public static final String DEFAULT_DISCARD_STREAM_ID = "druid-discard-stream";
+
+    //Tranquilizer configs for DruidBeamBolt
+    private int maxBatchSize;
+    private int maxPendingBatches;
+    private long lingerMillis;
+    private boolean blockOnFull;
+    private boolean enableDiscardStream;
```

Comment:
@omkreddy We do not need this as it was suggested in earlier comments. User would give only discardStreamId if discarded events need to be handled.